images/runtime/python/Dockerfile.template

```diff
@@ -9,7 +9,21 @@ ENV BUILD_NUMBER=${BUILD_NUMBER}
 # GOPATH is set to "/go" in the base image, so no need to set it again
 RUN ./build.sh python /opt/startupcmdgen/startupcmdgen
 
-FROM python:%PYTHON_BASE_IMAGE%
+FROM buildpack-deps:stretch AS main
+
+ENV PYTHON_VERSION %PYTHON_FULL_VERSION%
+
+# Add python binaries from Oryx base image
+COPY --from=mcr.microsoft.com/oryx/python-build-%PYTHON_VERSION%:%IMAGE_TAG% /opt /opt
+RUN set -ex \
+ && cd /opt/python/ \
+ && ln -s %PYTHON_FULL_VERSION% %PYTHON_VERSION% \
+ && ln -s %PYTHON_VERSION% %PYTHON_MAJOR_VERSION% \
+ && echo /opt/python/%PYTHON_MAJOR_VERSION%/lib >> /etc/ld.so.conf.d/python.conf \
+ && ldconfig
+
```

Comment:
Could you check if any of these are required? https://github.com/docker-library/python/blob/34c9df35e9a69e9f0edde88e861b543edb8bc07a/3.7/stretch/Dockerfile#L62

Also a problem with that official Dockefile is that it is not clear if the environment variables is for doing the building of the source or for anyone to consume when inheriting the docker image.